source/getting-started/index.rst

```diff
@@ -17,27 +17,28 @@ joining a project and working with your first instance.
 Step 1: Log in to Chameleon
 ===========================
 
-Good news: chances are that you won’t even need to create an account to log into
-Chameleon. This is because Chameleon supports federated login: if your
-institution is a member of `InCommon`_ (most US research and education
-institutions are) or if you have a Google account you can use a federated login.
-Otherwise, you will be guided to create an account. Our login pages will guide
-you through the process, or you can :ref:`read more about logging into Chameleon
-via federated login <federation>`.
+Just click the “Log in” button situated in the top right corner of `our main page
+<https://www.chameleoncloud.org>`_ -- you probably won’t even need to create an account! 
+
+If your institution is a member of `InCommon`_ (most US research and education
+institutions are) -- or if you have a Google account -- you can log in with your i
+nstitutional/Google credentials via the federated login. Otherwise, the log in process 
```

Comment:
```suggestion
institutions are) -- or if you have a Google account -- you can log in with your
institutional/Google credentials via the federated login. Otherwise, the log in process 
```